src/components/Firmware/Firmware.tsx

```
@@ -288,9 +309,12 @@ export default function Firmware() {
       case 'unknown':
         typeFlag = 'Unknown';
         break;
+      case 'classicpure':
+        typeFlag = 'OneKey Classic 1S Pure';
+        break;
       default:
         // eslint-disable-next-line no-case-declarations, no-unused-vars, @typescript-eslint/no-unused-vars
-        const checkType: never = originType;
+        const checkType = originType;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)